### ISSUE TITLE
[dataclasses] Implement pybindings for I3Particle::Clone() (Trac #2383)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2375]
see title

Migrated from https://code.icecube.wisc.edu/ticket/2383
```json
{
    "status": "closed", 
    "changetime": "2020-06-24T12:31:42", 
    "description": "see title", 
    "reporter": "chaack", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1593001902142004", 
    "component": "combo core", 
    "summary": "[dataclasses] Implement pybindings for I3Particle::Clone()", 
    "priority": "major", 
    "keywords": "", 
    "time": "2019-12-13T14:53:19", 
    "milestone": "Autumnal Equinox 2020", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20191213T16:44:28: 
* olivas changed owner from "" to "olivas"
* olivas changed status from "new" to "accepted"
* olivas changed milestone from "" to "Vernal Equinox 2020"

**Comment by icecube-trac** [comment]
Trac update at 20200217T19:52:32: olivas changed summary from "Implement pybindings for I3Particle::Clone()" to "[dataclasses] Implement pybindings for I3Particle::Clone()"

**Comment by icecube-trac** [comment]
Trac update at 20200303T21:35:01: 
* olivas changed status from "accepted" to "closed"
* olivas changed resolution from "" to "fixed"
* olivas commented:

> In changeset 179198:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="179198"
> added pybinding to I3Particle::Clone.  closes #2375.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20200624T12:31:42: 
* olivas commented:

> Milestone Vernal Equinox 2020 deleted


* olivas changed milestone from "Vernal Equinox 2020" to "Autumnal Equinox 2020"
